dev-python/PythonQt/PythonQt-3.1.ebuild

```diff
@@ -0,0 +1,89 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+
+inherit qmake-utils python-r1
```

Comment:
@SpiderX I totally missed the point of this package. I thought of it as a PyQt analogue. Anyhow, you need to go back to `python-single-r1`, as namespacing the .so files for different Python impls will be a nightmare, especially for revdeps.